### ISSUE TITLE
Error activating ModelMetadataProvider

### ISSUE THREAD

**Issue by maranganator** [lozanotek/mvcturbine#16]
When I add a ninject mvc turbine this error show up every time at the start in debug mode.

Error activating ModelMetadataProvider
No matching bindings are available, and the type is not self-bindable.
Activation path:
  1) Request for ModelMetadataProvider

Suggestions:
  1) Ensure that you have defined a binding for ModelMetadataProvider.
  2) If the binding was defined in a module, ensure that the module has been loaded into the kernel.
  3) Ensure you have not accidentally created more than one kernel.
  4) If you are using constructor arguments, ensure that the parameter name matches the constructors parameter name.
  5) If you are using automatic module loading, ensure the search path and filters are correct.

When I try to use structure map instead this error show up 

StructureMap Exception Code:  202
No Default Instance defined for PluginFamily System.Web.Mvc.ModelMetadataProvider, System.Web.Mvc, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35

**Comment by darrencauthon** [comment]
What version of MvcTurbine are you using?  I'm not able to replicate these issues with the latest version, with either Ninject or StructureMap.

**Comment by Necroskillz** [comment]
I confirm this issue with Ninject. Im using the latest version from NuGet (3.4). The exception shows up once, when the application starts, once you hit mvc templated helper in a view (like @Html.TextBoxFor<>()). You also have to be in debug mode.

I would really appreciate resolving this issue, since people on my team are constantly complaining about it.

**Comment by darrencauthon** [comment]
Ah, that helps -- I wasn't able to reproduce the issue, but I didn't test it against one of the MVC helpers.

I'll take a look tonight and see what I can find.  Sorry for the issue, I'm sure we can figure it out pretty quickly.

**Comment by darrencauthon** [comment]
Ok, I was able to reproduce the issue last night, but I haven't been able to resolve it.  I don't yet see the problem we're getting with Ninject versus some of the other IoC containers.  Still working...

**Comment by Necroskillz** [comment]
Hi, any progress?

**Comment by darrencauthon** [comment]
... sorry, haven't figured it out yet.  I took a quick look at it late last week, but I couldn't figure out what was going on with Ninject.  I'll take another look this week.  

Sorry -- I want to resolve this for you, but fighting these IoC containers takes a lot of energy.

**Comment by Necroskillz** [comment]
I noticed the issue is not there anymore when i tried the latest version from source control. Are you guys planning to push a new version to NuGet anytime soon?

**Comment by darrencauthon** [comment]
The issue is still there as far as I know, as I haven't fixed it.  Are you sure?

**Comment by Necroskillz** [comment]
Yeah, pretty sure. Try it yourself to be sure, but to me it seems that the problem just went away.
